scripts/generate-crd-schema.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ ! -d "schema" ]; then
+  mkdir schema
+fi
+
+convert_script=$(realpath scripts/openapi2jsonschema.py)
+crd_files=$(find ray-operator/config/crd/bases -name "*.yaml" -exec realpath {} \;)
+
+cd schema
+
+for crd_file in $crd_files; do
+  $convert_script $crd_file
```

Comment:
nit: My shellcheck VS Code extension is telling me to "Double quote to prevent globbing and word splitting"

```suggestion
  $convert_script "$crd_file"
```